aea/test_tools/click_testing.py

```diff
@@ -147,12 +148,12 @@ def invoke(  # type: ignore
         )
 
 
-class CliTest:
+class CliTest(ABC):
     """Test cli commands."""
 
-    t: Path
     cli_options: Sequence[str] = ()
 
+    _t: Path
```

Comment:
This is private to the test case. Test case expects it to not be altered (see below)